inference-engine/src/vpu/common/include/vpu/ngraph/operations/static_shape_reshape.hpp

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2020 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+
+#include "ngraph/op/util/attr_types.hpp"
+#include "ngraph/node.hpp"
+#include <ngraph/opsets/opset3.hpp>
+
+#include <memory>
+#include <vector>
+
+namespace ngraph { namespace vpu { namespace op {
+
+class StaticShapeReshape : public ngraph::opset3::Reshape {
```

Comment:
What about the `evaluate` method? Will you add it with tests impl?

Reply:
By default, I reuse evaluate method of original `Reshape`. I'm not sure it's enough, but I hope so. In case if test will require it - I'll add separate method